.github/workflows/scripts/pr_workflow.py

```diff
@@ -88,6 +88,8 @@ def __init__(self) -> None:
         self.user_login = os.getenv("GITHUB_USER_LOGIN")
         self.review_state = os.getenv("GITHUB_EVENT_REVIEW_STATE")
         self.comment_body = os.getenv("COMMENT_BODY", "")
+        if self.comment_body == "":
```

Comment:
please split; each action should have its relevant attributes and checks. 
so here, for example add `self.review_comment_body` etc

this helps to understand what is needed for each action, and it is easier to maintain different flows.
it is ok for  some flows share some things in common

Reply:
This was discussed further by me and Ruth; we agreed to keep overriding this variable but adding a comment explaining what's happening. Done now